extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaMediaInsertDialog.js

```diff
@@ -151,7 +152,12 @@ ve.ui.WikiaMediaInsertDialog.prototype.initialize = function () {
 
 	this.mainPage = new OO.ui.PageLayout( 'main', { $content: this.$mainPage } );
 	this.searchPage = new OO.ui.PageLayout( 'search', { $content: this.search.$element } );
-	this.pages.addPages( [ this.mainPage, this.searchPage ] );
```

Comment:
You can also just overwrite intialize method, make it call it parent method, and then undo what you "don't want", e.g.: ```this.pages.removePage(this.mainPage)```